### PR TITLE
Add bookworm target

### PR DESCRIPTION
**Pull request by cpuguy83**

### CODE REVIEW

File: distros_test.go
Comment (pmengelbert) [inline distros_test.go:1]: We're actually not using these files at all anymore. We're using the old tests, in the `tests` directory.

**Review by pmengelbert — CHANGES_REQUESTED** [review]
We don't need the test helper, since we're doing our tests using the old system.

File: distros_test.go
Comment (cpuguy83) [reply distros_test.go:1]: Yep, removed in an extra commit now.

**Review by pmengelbert — APPROVED** [review]
Thanks

**Pull request merged**